NRzure/alert-policies/example-alert-policy/baseline-alert.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
This appears to just be the placeholder alert condition, please remove it or fill it out with relevant information.